api/Pipfile

```diff
@@ -23,7 +23,7 @@ deepdiff = "*"
 Django = "==3.2.9"
 django-braces = "*"
 django-cors-headers = "*"
-django-oauth-toolkit = "==1.5.0"
+django-oauth-toolkit = "==1.7.0"
```

Comment:
@WordPress/openverse-developers anyone know why we pin this version in the Pipfile instead of just relying on the lock?

Reply:
It looks like it required some delicacy in the past due to associated migrations: https://github.com/WordPress/openverse-api/pull/67

I think this will also require some migrations. We could merge this in and deploy it with our API release later today!